exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/SqlSelectBuilder.java

```diff
@@ -90,8 +91,14 @@ public SqlSelectBuilder fetch(SqlNode fetch) {
     return this;
   }
 
+  public SqlSelectBuilder hints(SqlNodeList hints) {
```

Comment:
@vvysotskyi Will Drill do anything with the SQL hints that Calcite now supports?

Reply:
In the future, I think we will use this functionality to provide more flexibility for configuring the planning process.